client/packages/common/src/ui/components/inputs/TextInput/numeric/PositiveNumberInput.tsx

```diff
@@ -15,7 +16,8 @@ export const PositiveNumberInput = React.forwardRef(
       sx,
       disabled = false,
       value,
-      max = 999999999,
+      min = 1,
+      max = Number.MAX_SAFE_INTEGER,
```

Comment:
Can you revert this change please? It was the source of a bug previously, as noted.

Reply:
thanks done in:
https://github.com/openmsupply/open-msupply/pull/945